mongo-stream.js

```diff
@@ -48,9 +48,9 @@ class MongoStream {
       const collectionManager = new CollectionManager(this.db, collection, this.elasticManager);
       if (options.dump) {
         await collectionManager.elasticManager.deleteElasticCollection(collection);
-        await collectionManager.dumpCollection(options.ignoreResumeTokens);
```

Comment:
Now that we aren't using this as a parameter anymore we should take any sort of decision making out of the dumpCollection() function.